src/InvoiceGenerator.BusinessLogic/ClientService.cs

```diff
@@ -31,7 +31,7 @@ public List<ClientViewModel> GetClients()
         public List<ClientNameViewModel> GetClientNames()
         {
 
-            var clientNames = _invoiceDbContext.Clients.ToList();
+            var clientNames = _clientRepository.GetAll();
```

Comment:
:seedling: I have a small refactor in mind here; one which is more of an experience thing. No worries about making any changes right now, though